src/bin/command.rs

```diff
@@ -18,6 +18,9 @@ use dusk_wallet::gas::Gas;
 use dusk_wallet::{Address, Dusk, Lux, Wallet};
 use dusk_wallet_core::{BalanceInfo, StakeInfo};
 
+/// The default stake gas limit
+pub const DEFAULT_STAKE_GAS_LIMIT: i64 = 2_000_000_000;
```

Comment:
This value is actually too low. The `allow` function on the staking contract can have highly fluctuating gas spent costs associated with it, ranging from 1.4B LUX up to +2B LUX.

Observed example transactions:
https://explorer.dusk.network/transactions/transaction/?id=1ace07168147f2be166cc2feab7dc675b700220a5fdc8f2cdb698ad503ed8904
https://explorer.dusk.network/transactions/transaction?id=7c2c1e3f1dd2a4abec05d6ffd037acda351eb7f36dc110108b060bd94b48ff07
https://explorer.dusk.network/transactions/transaction?id=eef868d4a7eb204b5d38c591aaeea385658670566d5b2f44ccbd0ce846bf6705
https://explorer.dusk.network/transactions/transaction?id=0c2f95eac535279e79338190787f7270735f670dca78804d1568d902c8053a0d
https://explorer.dusk.network/transactions/transaction?id=219a8c0b1dbe7771d2b5976a5e6f39e16c1444f3cf890b575f6ea6e4e5372604